src/config/constants/farmAuctions.ts

```diff
@@ -1,130 +1,182 @@
+import { Token, Pair, ChainId } from '@pancakeswap/sdk'
+import { WBNB, BUSD } from './tokens'
 import { FarmAuctionBidderConfig } from './types'
 
+const getLpAddress = (token0: string, token1: Token) => {
+  const token0asToken = new Token(ChainId.MAINNET, token0, 18)
+  return Pair.getAddress(token0asToken, token1)
+}
+
 export const whitelistedBidders: FarmAuctionBidderConfig[] = [
   {
     account: '0x9Ed5a62535A5Dd2dB2d9bB21bAc42035Af47F630',
     farmName: 'NAV-BNB',
+    tokenAddress: '0xbfef6ccfc830d3baca4f6766a0d4aaa242ca9f3d',
+    quoteToken: WBNB,
     tokenName: 'Navcoin',
     projectSite: 'https://navcoin.org/en',
   },
   {
     account: '0x33723811B0FCa2a751f3912B80603Fe11499D894',
     farmName: 'WSG-BNB',
+    tokenAddress: '0xa58950f05fea2277d2608748412bf9f802ea4901',
+    quoteToken: WBNB,
     tokenName: 'Wall Street Games',
     projectSite: 'https://wsg.gg/',
   },
   {
     account: '0xD1C35C3F5D9d373A3F7c0668Fbe75801886e060f',
     farmName: 'SWIRGE-BNB',
+    tokenAddress: '0xe792f64C582698b8572AAF765bDC426AC3aEfb6B',
+    quoteToken: WBNB,
     tokenName: 'Swirge Network',
     projectSite: 'https://swirge.com/',
   },
   {
     account: '0x58092273a044D6e1d23B5095AE873F6E24E906ed',
     farmName: 'rUSD-BUSD',
+    tokenAddress: '0x07663837218a003e66310a01596af4bf4e44623d',
+    quoteToken: BUSD[ChainId.MAINNET],
```

Comment:
Maybe look at creating a `getAddress` helper for `Token` entity (same as for `Address` entity); unless doing the above which wouldn't be necessary